src/Admin/BreadcrumbsBuilderInterface.php

```diff
@@ -30,7 +30,7 @@ interface BreadcrumbsBuilderInterface
      * @param string $action the name of the action we want to get a
      *                       breadcrumbs for
      *
-     * @return iterable the breadcrumbs
+     * @return ItemInterface[] the breadcrumbs
```

Comment:
We don't actually require implementations to return an array, do we?

```suggestion
     * @return iterable<ItemInterface> the breadcrumbs
```

Reply:
`BreadcrumbsBuilder::getBreadcrumbs` returns an array. There is no usage of `yields` in our code, so we should use simple arrays.

Reply:
That should not be dictated by our implementations, but by our usage in the client code, which doesn't require the result to be an array AFAIK